client/src/views/files/FoldersPage.vue

```diff
@@ -307,7 +295,10 @@ const tabBarActions = computed(() => {
     });
   }
   if (selectedEntries.length === 1 && selectedEntries[0].isFile()) {
-    actions.push({ label: 'FoldersPage.tabbar.open', action: async () => await openEntries(getSelectedEntries()), icon: open });
+    actions.push(
+      { label: 'FoldersPage.tabbar.preview', action: async () => await openEntries(getSelectedEntries()), icon: eye },
+      { label: 'FoldersPage.tabbar.edit', action: async () => await editEntries(getSelectedEntries()), icon: create },
```

Comment:
Needs to switch the icon to `RenameIcon`. Some changes could need to be done in `TabBarOptionsModal.vue` and `TabBarOptions.vue` to use `MsImage` instead of `IonIcon`.